jwst/rscd/rscd_sub.py

```diff
@@ -249,8 +249,8 @@ def get_rscd_parameters(input_model, rscd_model):
         log.warning('No matching row found in RSCD reference table for')
         log.warning('READPATT=%s, SUBARRAY=%s, Row type=ODD',
                     readpatt, subarray)
-        raise NoTableEntry('No match for READPATT or SUBARRAY in RSCD ref table')
-        return None, None
+
```

Comment:
Should add the same warning here as in line 220, "RSCD step will be skipped", before returning.